skillsParser/models/user.py

```diff
@@ -77,13 +77,10 @@ def __create_user(self, engine):
                        skills_details=self.skills,
                        previous_experience=self.previous_expr)
 
-        result = engine.execute(text("""SELECT user_id FROM users WHERE email = :email LIMIT 1"""), email=self.email)
-        row = result.fetchone()
-        user_id = row.values()[0]
-        return user_id
+        return self.email
 
 
-    def __create_user_languages(self, engine, user_id):
+    def __create_user_languages(self, engine, user_email):
```

Comment:
yeah, no 2 people should ever have the same email, so user_id is silly.  I made the user's pkey the user's email since we require that field.